package.json

```diff
@@ -48,10 +48,10 @@
     "prettier:check": "prettier --check 'packages/**/*.{ts,css,md,sol}'",
     "prettier": "prettier --write 'packages/**/*.{ts,css,md,sol}'",
     "lint": "pnpm prettier:check && eslint . --ext .ts",
-    "release": "pnpm install && lerna publish --no-private --force-publish && pnpm retype:updateversion",
-    "release:ci": "lerna publish --no-private --force-publish --yes",
-    "release:canary": "lerna publish premajor --canary --no-private --force-publish",
-    "release:manual": "pnpm --recursive --no-bail run release",
+    "release": "pnpm recursive run clean && pnpm install && lerna publish --no-private --force-publish && pnpm retype:updateversion",
```

Comment:
not directly related to this PR, but before #623 we had a build step before `lerna publish` (implicit via the prepare script) to double check the build succeeds before attempting to publish. The reason for this was that otherwise lerna already created tags etc even if later building one of the packages failed, resulting in a messy manual clean up process.

I'm not sure if this is still the case with the current lerna version, but would prefer to re-add that build step to be sure (unless you know a better way to prevent this)

Also small nit: should we create a `clean`/`clean:forge` script and invoke it here to make these scripts slightly less verbose?